internal/web/datasethandler.go

```diff
@@ -250,6 +252,10 @@ func (handler *datasetHandler) getEntitiesHandler(c echo.Context) error {
 	}
 
 	f := c.QueryParam("from")
+	reverse := c.QueryParam("reverse")
+	if reverse != "" {
+		return echo.NewHTTPError(http.StatusBadRequest, fmt.Errorf("reverse parameter only supported for changes"))
```

Comment:
For Echo, you want to return text, not the err, it has to do with the pathing of the return values in Echo, a return echo.NewHttpError(500, err.Error()) returns the Echo error object, while the echo.NewHttpError(500, err) does not.